astroplan/tests/test_exptime.py

```diff
@@ -0,0 +1,190 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+# Third-party
+import astropy.units as u
+from astropy.tests.helper import assert_quantity_allclose
+import numpy as np
+from astroplan.exceptions import UserInputError
+from astropy.tests.helper import raises
+import astropy.units as u
+import numpy as np
+from astroplan import FixedTarget, Observer, Telescope
+from astroquery.gaia import Gaia
```

Comment:
Can you please move this line to the inside of the test function that calls the `Gaia` module? It's part of the reason why the tests are failing.